test/integration/base.py

```diff
@@ -28,6 +29,8 @@ def __init__(self):
 
 class DBTIntegrationTest(unittest.TestCase):
 
+    prefix = "test{}".format(random.randint(0, 999))
```

Comment:
can we make this something else? possible collision rate here is too close for comfort.